website/docs/getting-started/cluster-setup/kubernetes.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Kubernetes
+sidebar_position: 1
+---
+
+# Setup a Kubernetes cluster
```

Comment:
We should be consistent in the form used for the titles: **Setup a Kubernetes cluster** vs **Setting up a Kubernetes cluster**. The second form seems more appropriate to me.